ui/oopsyraidsy/buff_map.ts

```diff
@@ -34,8 +36,9 @@ export const missedEffectBuffMap: readonly MissableEffect[] = [
     collectSeconds: 15,
   },
   {
+    // PLD
     id: 'Divine Veil',
-    type: 'mitigation',
+    type: 'heal',
```

Comment:
```suggestion
    type: 'mitigation',
```

Yeah it's also a heal, but I think most people use it as mitigation.

Reply:
Same comment as above.

Reply:
It might make sense to add `shield` as a `MissableBuffType` instead.